pkg/test-infra/fixture/fixture.go

```diff
@@ -170,12 +170,12 @@ func init() {
 	flag.StringVar(&Context.LokiPassword, "loki-password", "", "loki password. Needed when basic auth is configured in loki")
 
 	flag.StringVar(&Context.ImageVersion, "image-version", "latest", "image version")
-	flag.StringVar(&Context.TiDBConfigFile, "tidb-configmap", "", "path of tidb configmap (cluster A in abtest case)")
-	flag.StringVar(&Context.TiKVConfigFile, "tikv-configmap", "", "path of tikv configmap (cluster A in abtest case)")
-	flag.StringVar(&Context.PDConfigFile, "pd-configmap", "", "path of pd configmap (cluster A in abtest case)")
-	flag.StringVar(&Context.ABTestConfig.TiDBConfigFile, "abtest.b.tidb-configmap", "", "tidb configmap for cluster b")
-	flag.StringVar(&Context.ABTestConfig.TiKVConfigFile, "abtest.b.tikv-configmap", "", "tikv configmap for cluster b")
-	flag.StringVar(&Context.ABTestConfig.PDConfigFile, "abtest.b.pd-configmap", "", "pd configmap for cluster b")
+	flag.StringVar(&Context.TiDBConfigFile, "tidb-config", "", "path of tidb config file (cluster A in abtest case)")
+	flag.StringVar(&Context.TiKVConfigFile, "tikv-config", "", "path of tikv config file (cluster A in abtest case)")
+	flag.StringVar(&Context.PDConfigFile, "pd-config", "", "path of pd config file (cluster A in abtest case)")
+	flag.StringVar(&Context.ABTestConfig.TiDBConfigFile, "abtest.b.tidb-config", "", "tidb config file for cluster b")
```

Comment:
And how about rename `abtest.b.xxx` as `abtest.xxx`? eg, we use tidb-config as `abtest.a.tidb-config` in fact, so I think we can remove the `b.` prefix here?

Reply:
Yes, sounds reasonable.